lib/tasks/manageiq/ui_tasks.rake

```diff
@@ -87,27 +87,33 @@ namespace :webpack do
   end
 end
 
+miq_app_prefix = defined?(ENGINE_ROOT) ? "app:" : ""
+
 # compile and clobber when running assets:* tasks
-if Rake::Task.task_defined?("assets:precompile")
-  Rake::Task["assets:precompile"].enhance do
-    Rake::Task["webpack:compile"].invoke unless ENV["TRAVIS"]
+if Rake::Task.task_defined?("#{miq_app_prefix}assets:precompile")
+  unless ENV["TRAVIS"]
+    Rake::Task["#{miq_app_prefix}assets:precompile"].enhance do
+      Rake::Task["webpack:compile"].invoke
+    end
   end
 
-  Rake::Task["assets:precompile"].actions.each do |action|
+  Rake::Task["#{miq_app_prefix}assets:precompile"].actions.each do |action|
```

Comment:
I'm not sure this is a valid change.

This code should undo webpacker addin those actions to `assets:precompile`. Webpack never adds it to `app:` anything.